### PR TITLE
NL-SfB classification in accordance with the BIM Basic ILS tile 3.6.

### PR DESCRIPTION
**Pull request by nnijmeijer**
Following the "discussion" in Issues "Add custom skc #14" I made some changes to the code and the NL-SfB classification scheme.
Note: The NL-SfB classification scheme is the 2005 version.

Thanks to these changes it is possible to export the NL-SfB classification name in accordance with the BIM Basic ILS tile 3.6.

### CODE REVIEW

File: src/bt_ifcmanager/lib/lib_ifc/IfcProduct_su.rb [hunk 1]
```diff
@@ -213,7 +210,7 @@ def create_representation(faces, transformation, su_material)
       end
       
       # add color from su-object material, or a su_parent's
-      if @ifc_model.options[:colors]
+      if @ifc_model.options[:styles]
```
Comment (janbrouwer) [inline src/bt_ifcmanager/lib/lib_ifc/IfcProduct_su.rb:213]: Is this change on purpose? A couple of months ago I changed the 'styles' option to 'colors' to better align with the sketchup terminology. If you think this is a bad change than it's ok to revert...

Comment (nnijmeijer) [reply src/bt_ifcmanager/lib/lib_ifc/IfcProduct_su.rb:213]: Clear name change.

I had changed it back during a test with an "older" version.
